generators/app/package-json.js

```diff
@@ -229,7 +229,7 @@ const packageJSON = (current, context) => {
 
   if (context.css) {
     _.assign(result.scripts, {
-      'build:css': scriptify('postcss --verbose -o dist/%s.css --config scripts/postcss.config.js src/plugin.css'),
+      'build:css': scriptify('postcss -o dist/%s.css --config scripts/postcss.config.js src/plugin.css'),
```

Comment:
the verbose output for postcss cli is basically useless it just prints the file that was processed in color and nothing else.